src/world/physicsReferences.lua

```diff
@@ -1,21 +1,35 @@
 local PhysicsReferences = {}
 
---Contact Filter references
---Fixture Categories
+--[[ Types
```

Comment:
Could you reformat this explanatory comment like this:

```lua
--- Summary
-- More explanation that can span as many lines as you like.
```

If you format it like this then it will show up in our documentation if we start using LDoc.